### ISSUE TITLE
[Feature Request] Add a library filter "has duplicate chapters"

### ISSUE THREAD

**Issue by isabella-almeidaa** [Suwayomi/Suwayomi-WebUI#777]
## What feature should be added to Suwayomi?
Just a filter for the library that shows which entries has duplicate chapters

##  Why/Project's Benefit/Existing Problem
Because duplicate chapters usually create a lot of clutter and confusion while reading. This would be a way to know which entries actually has duplicate chapters without having to look on all of the entries of your library, and then filter the duplicate chapters out.

**Comment by schroda** [comment]
I don't quite get the problem this would solve.
do you have multiple manga from different sources in your library and only want to read from the ones which don't have multiple scanlators?
if so, why add the ones with multiple scanlators to your library in the first place?

`and then filter the duplicate chapters out.`
how does this connect to being able to filter the library based on duplicated chapters, sounds more like you want to be able to filter the chapter list based on scanlators?

`Because duplicate chapters usually create a lot of clutter and confusion while reading`
can you give a examples of what you mean?
does the `skip duplicated chapters` settings not work in these cases?

**Comment by isabella-almeidaa** [comment]
> I don't quite get the problem this would solve. do you have multiple manga from different sources in your library and only want to read from the ones which don't have multiple scanlators? if so, why add the ones with multiple scanlators to your library in the first place?

The problem I want to solve is having to check out the entry from the first chapter to the last to see if there's any duplicates in between. If a filter was added, I would know exactly the ones I would have to check and filter out so I can start reading without problems
Most of the sources I use are aggregators, so the manga I read usually have multiple duplicate chapters. Adding ones that doesn't have multiple chapters isn't an option in some cases because most of the time I won't even find the manga I'm looking for, the source doesn't have the chapters of the previous source or it will sitll have duplicate chapters... If the source I'm using has multiple chapters, I'll have to filter them. What I usually do is compare both chapters so I can choose which scanlator I'll filter out (because of translation/image quality)

> `and then filter the duplicate chapters out.` how does this connect to being able to filter the library based on duplicated chapters, sounds more like you want to be able to filter the chapter list based on scanlators?

No.. what I want is the filter in the library so I'll be able to know which entries I have to go through to filter out (what I usually do to filter out is bookmarking the chapters that I want to filter out)

> `Because duplicate chapters usually create a lot of clutter and confusion while reading` can you give examples of what you mean? does the `skip duplicated chapters` settings not work in these cases?

Basically, If I'm starting to read a new manga, I won't be able to just click start. I'll have to check out if it doesn't have duplicate chapters, and then I'll start reading it. It creates clutter because there will be multiple chapters one after another that are duplicates, or like the example from before, it'll have multiple chapters, let's say, 1-12, which are scanlated chapters. Then the previous scanlator stopped because the official chapters are starting to come out and then an official chapter uploader will upload everything from the start.
Also, the "skip duplicated chapters" setting wouldn't work because I won't be able to decide which scanlator I wish to read from.

**Comment by schroda** [comment]
I see, this will require server changes to be usable, otherwise, loading the required data and filtering it would be too inefficient

**Comment by isabella-almeidaa** [comment]
> I see, this will require server changes to be usable, otherwise, loading the required data and filtering it would be too inefficient

should I make an issue on the server repo?
Although idk which would be the changes necessary so I wouldn't know what to put as a title...
